hubstorage/serialization.py

```diff
@@ -1,4 +1,5 @@
 from json import dumps, loads
+from msgpack import Unpacker
```

Comment:
this is not backwards compatible because introduces new dependency. I know that in Scrapy it's usually done as a check - if package is installed - we use some advanced feature - if not - we fallback to what works. possibly we need some @dangra wisdom here...